int8 portDrive = 0
int8 starboardDrive = 1
int8 depWinch = 2
int8 bcBeltDrive = 3
int8 bcTranslation = 4
int8 bcAttitude = 5
int8 portLooky = 6
int8 starboardLooky = 7
int8 bcLED = 8

int8 motorID
float64 value

